src/Microsoft.ML.Data/Prediction/PredictionEngine.cs

```diff
@@ -103,7 +103,7 @@ public abstract class PredictionEngineBase<TSrc, TDst> : IDisposable
         /// <summary>
         /// Provides output schema.
         /// </summary>
-        public DataViewSchema OutputSchema;
+        public DataViewSchema OutputSchema { get; }
```

Comment:
A mutable schema on the base class of the prediction engine. *That* would have been embarrassing. :)